_layouts/item.html

```diff
@@ -13,47 +13,45 @@ <h1 class="page__title item__title">
     <p class="item__callnumber">Call Number: {% include call_number.html object=object %}</p>
   </div>
   <div class="item__header-right">
-    <button class="btn btn--md btn--blue" data-micromodal-trigger="request_modal">
+    <button class="btn btn--blue btn--md" data-micromodal-trigger="modal__date">
       Request in Reading Room
     </button>
   </div>
 </div>
 
-<div class="item-details">
-  <div class="wrapper--flex-wrap wrapper--attributes">
-    <div class="container item__attribute">
-      <h2 class="item-details__label">Authors</h2>
-      {%- capture authors -%}{%- include author_list.html object=object separator="|" -%}{%- endcapture -%}
-      {%- assign authors = authors | split: "|" -%}
-      <ul class="list--unstyled">
-      {% for author in authors %}
-        <li>{{author}}</li>
-      {% endfor %}
-      </ul>
-    </div>
-    {%- capture subjects -%}{%- include subject_list.html object=object separator="|" -%}{%- endcapture -%}
-    {% if subjects.size > 0 %}
-    {% assign subjects = subjects | split: "|" | uniq %}
-    <div class="container item__attribute">
-      <h2 class="item-details__label">Subject Terms</h2>
-      <ul class="list--unstyled">
-      {%- for subject in subjects -%}
-        <li>{{subject}}</li>
-      {%- endfor -%}
-      </ul>
-    </div>
-    {% endif %}
-    <div class="container item__attribute">
-      <h2 class="item-details__label">Published</h2>
-      {% include publication_info.html object=object %}
-    </div>
+<div class="grid">
+  <div class="p-10" style="grid-column: 1 / span 4">
```

Comment:
To keep things maintainable and clear, instead of using inline styles in the HTML you'll want to define a CSS class here, and then use that class name in `custom.css` to define the styles.